Terminal.Gui/Application.cs

```diff
@@ -1631,6 +1648,48 @@ bool AdornmentHandledMouseEvent (Adornment frame)
                 BringOverlappedTopToFront ();
             }
         }
+
+        bool AdornmentViewHandledMouseEvent ()
```

Comment:
I'll rename it to `AdornmentSubViewHandledMouseEvent`. I think these logic could be moved to View class and only return the value.

Reply:
> I'll rename it to `AdornmentSubViewHandledMouseEvent`. I think these logic could be moved to View class and only return the value.

I only renamed the method. About the possibility to move to the `View` class I leave that to you, because it access private fields on the `Application` class and I only added this method like you already did before with the others local methods. I doubt they needed to be called from another location than this.
Done.

Reply:
> I'll rename it to `AdornmentSubViewHandledMouseEvent`. I think these logic could be moved to View class and only return the value.

See my comment above. As much as possible, any logic specific to Adornments should be in the Adornment class or classes that derive from it. You should not move it to View, if you can avoid doing that.